src/scripts/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+window.addEventListener('hashchange', () => {
+  if (window.location.hash === '#menu') {
+    document.body.classList.add('page__body--with-menu');
+  } else {
+    document.body.classList.remove('page__body--with-menu');
+  }
+});
+
+const defaultSubmit = document.querySelector('.contacts__field-button');
+
+defaultSubmit.addEventListener('click', addDefaultValue);
+
+function addDefaultValue(event) {
+  event.preventDefault();
+}
```

Comment:
Validation doesn't work.
I think the problem could be here.
Maybe you can try to pick a `form` element and add addEventListener to it on` 'submit'` - instead of on click by button.

#13 Placeholders in the forms suggest what to enter; apply validation of the form fields (required, email / tel etc.), then it is clear in what format to enter the data